### PR TITLE
add new step to generate the translation strings

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

**Comment by cpanato** [comment]
@jasonblais for now yes, i know @jespino need to fix some things in the server/enterprise repo side first

I will add

File: site/content/contribute/webapp/developer-workflow.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ Here's a general workflow a Mattermost developer working on the [mattermost-weba
...
-5. Once everything works to meet the ticket requirements, stop Mattermost by running `make stop` in the server repository, then run `make check-style` to check your syntax and `make test` to run the tests.
-6. Commit your changes, push your branch and [create a pull request](https://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request).
-7. Respond to feedback on your pull request and make changes as necessary by committing to your branch and pushing it. You might need to [rebase your changes](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) if another pull request creates conflicts.
-8. That's it! Rejoice that you've helped make Mattermost better.
+5. If you added or changed any strings you will need to run `make i18n-extract`to generate the new/updated strings.
+6. Once everything works to meet the ticket requirements, stop Mattermost by running `make stop` in the server repository, then run `make check-style` to check your syntax and `make test` to run the tests.
+7. Commit your changes, push your branch and [create a pull request](https://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request).
```
Comment (amyblais) [inline site/content/contribute/webapp/developer-workflow.md]: This link is broken. Maybe link here https://developers.mattermost.com/blog/submitting-great-prs/?

Comment (cpanato) [reply site/content/contribute/webapp/developer-workflow.md]: i can fix that, not part of this pr but i can fix

Comment (cpanato) [reply site/content/contribute/webapp/developer-workflow.md]: done

**Comment by cpanato** [comment]
@jasonblais ptal

**Pull request merged**